src/community/mod.rs

```diff
@@ -110,6 +163,14 @@ impl Community {
         }
     }
 
+    pub fn add_addon(&mut self, addon_config: CommunityAddOnConfigId) {
+        self.addon_list.push(addon_config);
+    }
+
+    pub fn remove_addon(&mut self, config_id: String) {
+        self.addon_list.retain(|config| config != &config_id);
+    }
```

Comment:
How do you envision the layout of addons on the community page (on UI)? If we display them as tabs, there should be a way to order them, so instead of removing them, we may have `set_addons` where the whole list of addon configs will be expected.